test/network/setup/config.js

```diff
@@ -73,7 +73,7 @@ const config = {
 
 		// Configuring nodes to forge with force or without
 		const delegatesMaxLength = Math.ceil(
-			devConfig.forging.delegates.length / configurations.length
+			devConfig.forging.delegates.length / (configurations.length - 1)
```

Comment:
That's wrong, just for the purpose of Peer Disconnect and Blacklists we shouldn't change the network setup for all of the remaining tests.